src/App.jsx

```diff
@@ -14,70 +15,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {
+          selectedGood
+            ? `${selectedGood} is selected`
+            : 'No goods selected'
+        }
+        {
+          selectedGood
+            && (
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => setSelectedGood('')}
+              />
+            )
+        }
+      </h1>
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      <table className="table">
+        <tbody>
+          {
+            goods.map(good => (
+              <tr
+                data-cy="Good"
+                key={goods.id}
```

Comment:
What is `goods.id`? `goods` is an array of strings
```suggestion
                key={good}
```